modules/side-panel/react/lib/SidePanel.tsx

```diff
@@ -52,13 +52,13 @@ const SidePanelContainer = styled('div')<SidePanelProps>(
     position: 'absolute',
   },
   ({open}) => ({
-    alignItems: open ? '' : 'center',
-    boxShadow: !open ? '0 8px 16px -8px rgba(0, 0, 0, 0.16)' : '',
+    alignItems: !open ? 'center' : undefined,
```

Comment:
Don't have a strong opinion on this, but I wonder if it is not better to use `alignItems: open ? undefined : 'center',` instead of `!open`.